reorganize.py

```diff
@@ -0,0 +1,169 @@
+# reorganize.py
+# this program takes the output of the outbreak simulator and converts it into a csv file with relevant data shown in tabular format 
+# average values are given first, and individual results for each parameter for each simulation are given
+
+#imports needed libraries
+import numpy as np
+import pandas as pd
+import re
+import matplotlib.pyplot as plt
+import argparse
+
+def reorganize(file):
+    def createDictionary(txtfile): #converts txt file to dictionary
+        with open(txtfile) as f:
+            pq_lines = f.readlines()
+
+        for i in range(0, len(pq_lines)):
+            split = pq_lines[i].split("\t")
+            key = split[0]
+            value = split[1]
+            pq_dict[key] = value
+        print('Dictionary Created')
+
+        # delete keys that can't be turned into table form
+        del_list = [
+        'logfile',
+        'popsize',
+        'handle_symptomatic',
+        'interval',
+        'prop_asym_carriers',
+        'leadtime',
+        'n_simulation',
+        'total_infection',
+        'total_infection_failed',
+        'total_infection_avoided',
+        'total_infection_ignored',
+        'total_show_symptom',
+        'total_removal',
+        'total_recover',
+        'total_quarantine',
+        'total_reintegration',
+        'total_abort',
+        'total_asym_infection',
+        'total_presym_infection',
+        'total_sym_infection',
+        'n_remaining_popsize_3499',
+        'n_no_outbreak',
+        'n_no_infected_by_seed',
+        'n_seed_show_no_symptom',
+        'n_no_first_infection',
+        'n_first_infection_on_day_1',
+        'n_first_infection_on_day_2',
+        'n_first_symptom',
+        'n_first_symptom_on_day_1',
+        'n_second_symptom',
+        'n_second_symptom_on_day_1',
+        'n_third_symptom',
+        'n_third_symptom_on_day_1',
+        ]
+
+        for i in del_list:
+            del pq_dict[i]
+        print('Extraneous Deleted')
+    def cellFill(key): #fills in cells given dict key; exception case checking for n_outbreak_duration because it has a list of 1 element

```

Comment:
Formatting is bad. Please install vscode and set up `yapf` as formatter (or run `yapf` directly). You can also install `isort` and `pyflakes` to help you improve source code.